catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java

```diff
@@ -496,11 +498,23 @@ private String updateColumnPositionFieldDefinition(
     return columnDefinition.toString();
   }
 
-  private String deleteColumnFieldDefinition(TableChange.DeleteColumn deleteColumn) {
+  private String deleteColumnFieldDefinition(
+      TableChange.DeleteColumn deleteColumn, CreateTable lazyLoadCreateTable) {
     if (deleteColumn.fieldName().length > 1) {
       throw new UnsupportedOperationException("Mysql does not support nested column names.");
     }
     String col = deleteColumn.fieldName()[0];
+    boolean colExists =
+        lazyLoadCreateTable.getColumnDefinitions().stream()
+            .map(MysqlTableOperations::getColumnName)
+            .anyMatch(s -> StringUtils.equals(col, s));
+    if (!colExists) {
+      if (BooleanUtils.isTrue(deleteColumn.getIfExists())) {
+        return "";
```

Comment:
Does MySQL support `ALTER TABLE table_name` syntax if we get a empty string here directly?

Reply:
suggest to add a check to skip execute if  Alter SQL content is empty.